devel/testnet/README.KR.md

```diff
@@ -0,0 +1,118 @@
+# 노드 운영 방법 및 조건 
+
+보스아고라 노드인 아고라(Agora)를 테스트하는데 필요한 단계와 테스트 네트워크(‘TestNet’)에서 Foundation 노드와 상호 작용하는 방법을 안내드립니다.
+
+## 검증자(Validator) VS 풀 노드(Full Node)
+
+아고라는 전체노드(Full Node) 또는 검증자(Validator) 두 가지의모드 중 하나로 실행할 수 있습니다. 검증자를 실행하기 전에 전체 노드를 실행하는 방법을 익히는 것이 좋습니다. 잘못 구성하면 네트워크가 불안정해지고 펀드가 삭감되어 검증자를 실행할 수 없게 될 수 있습니다.
+
+검증자는 인터넷에서 공개적으로 접근 가능해야 하므로 구성이 더 복잡합니다. 전체노드에 적용되는 모든 지침은 검증자에도 적용됩니다. 이 안내서에서 ‘전체노드(Full Node)’는 단순히 ‘노드(Node)'라고 부릅니다.
+
+## 시스템 요구 사항
+
+다음은 노드를 실행하기 위한 최소 요구 사항입니다:
+- x86_64 프로세서 또는 호환 가능한, > 2GHz
+- 4GB RAM
+- 1GB의 저장 용량
+- Linux, Mac,Windows에 Docker 또는 WSL
+- 인터넷 연결
+
+2010년 이후에 제작된 대부분의 컴퓨터는 이러한 요구 사항을 충족될 것입니다.
+
+## Docker 또 native binaries
+
+Agora 실행 권장 방법은 Docker 이미지를 사용하는 것입니다.
+Ubuntu Linux 20.04 및 MacOS-11(Big Sur)용 네이티브 바이너리(Native Binaries)도 사용할 수 있습니다. 또한 설명서를 사용하여 아고라를 자신의 기계로 컴파일할 수도 있습니다.
+
+Docker 사용자의 경우 Docker가 아직 설치되지 않은 경우 [여기로](https://github.com/bosagora/agora#build-instructions) 이동하셔서 설치하시면 됩니다.
+아고라 다운로드는 첫 실행 시 자동으로 이루어집니다.
+
+네이티브 바이너리의 경우 [Agora의 릴리스 페이지로](https://github.com/bosagora/agora/releases) 이동하여 시스템과 일치하는 바이너리를 가져오시면 됩니다.
+예를 들어, 이 기록의 마지막 릴리스는 `v0.35.0` 입니다. 시스템과 일치하는 `agora` 바이너리를 찾으세요.
+
+![Asset list for v0.33.0](./Release.v0.33.0.png)
```

Comment:
Could you provide translation for those ? They are used as alt text for images (e.g. if the browser can't download the image, or for accessibility).